docs/index.asciidoc

```diff
@@ -0,0 +1,209 @@
+:plugin: dissect
+:type: filter
+
+[id="plugins-{type}-{plugin}"]
+
+=== Dissect
+
+include::plugin_information.asciidoc[]
+include::../../../logstash/docs/include/plugin_header.asciidoc[]
```

Comment:
@dedemorton I've thought about this,  most of our doc and static pages repo are in the Logstash [repo](https://github.com/elastic/logstash/tree/master/docs), The [logstash-docs](https://github.com/elastic/logstash-docs) repo for me was always a step to make the whole publish job work but we didn't actually commit docs there, am I wrong?